specs/AgriFood/AgriFarm/schema.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://fiware.github.io/dataModels/specs/AgriFood/AgriFarm/schema.json",
+    "title": "FIWARE - Agri Farm",
+    "description": "This entity contains a harmonised description of a generic farm made up of buildings and parcels. This entity is primarily associated with the agricultural vertical and related IoT applications.",
+    "type": "object",
+    "allOf": [
+        { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+        { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons" },
+        {
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": ["AgriFarm"],
+                    "description": "NGSI Entity Type"
+                },
+                "landLocation": {
+                    "$ref": "http://json-schema.org/geojson/geometry.json#",
+                    "description": "Geometry defining the boundaries of the farm"
+                },
+                "contactPoint": {
+                    "type": "object",
+                    "description": "Contact information of the farm i.e email, telephone, etc"
```

Comment:
indentation. please reference schema.org

Reply:
address is missing. please check consistency of the schema.

Reply:
I didn't include address as is defined in location commons along with the location attribute

Reply:
very true!